docs/notebooks/demo.ipynb

```diff
@@ -77,7 +100,10 @@
    "metadata": {},
    "source": [
     "\n",
-    "Additionally, we need to define functions that represent the action space and function(s) to compute reward. Because we consider an output of `T` a success and an output of `F` a failure, we'll have our reward function return the value of the `T` column of our `clf.classes_` array. "
+    "Additionally, we need to define functions that represent the action space and function(s) to compute the reward. Because we consider an output of `C` a success and an output of `N` a failure, we'll have our reward function return the value of the `C` column of our `clf.classes_` array. The `.classes_` won't be initialized until the learner learns from the data at least once, but you can see the order of the classes in `clf.alphas.keys()`\n",
+    "\n",
+    "*Aside: [as defined in the `Arm` class](https://github.com/bayesianbandits/bayesianbandits/blob/8378f2566153285a224f42ca9667e2d22f9f472b/bayesianbandits/_arm.py#L108), a reward function takes samples from the posterior as input: `reward_function(self.learner.sample(X, size))`*\n",
```

Comment:
Might be worth linking the type signature here: https://github.com/bayesianbandits/bayesianbandits/blob/8378f2566153285a224f42ca9667e2d22f9f472b/bayesianbandits/_arm.py#L15

A reward function takes samples from the posterior as input and returns either a float or an array of floats.

Reply:
Done!